src/Ryujinx.Ava/Modules/Updater/Updater.cs

```diff
@@ -314,6 +314,17 @@ private static async void UpdateRyujinx(Window parent, string downloadUrl)
 
                         executablePath = "/bin/bash";
                         arguments.InsertRange(0, new List<string> { updaterScriptPath, baseBundlePath, newBundlePath, currentPid });
+                        Process.Start(executablePath, arguments);
```

Comment:
`executablePath` don't seems to be used elsewhere now. Maybe you could remove the above condition and related code too and then do:

```suggestion
                        Process.Start("/bin/bash", arguments);
```

Reply:
I cleaned up the code a bit, removing some variables that aren't really being used anymore.